packages/common/src/util/sleep.ts

```diff
@@ -8,6 +6,6 @@ import { promisify } from "util";
  * on failure, and this function will sleep the same amount of time every time
  * the test is retried.  Prefer {@link sleepWithBackoff} instead.
  */
-const sleep = promisify(setTimeout);
-
-export default sleep;
+export default function sleep(timeout: number): Promise<void> {
```

Comment:
util import is not available in docusaurus. It was either changing this or installing a polyfill